usdt-impl/src/stap3.rs

```diff
@@ -0,0 +1,184 @@
+//! The SystemTap probe version 3 of the USDT crate.
+//!
+//! Used on Linux platforms without DTrace.
```

Comment:
I think we usually put this below the copyright and license stuff?

Reply:
Fixed.